requirements.txt

```diff
@@ -1,5 +1,4 @@
 numpy~=1.19
 hdmf~=2.5
 pynwb~=1.4
-ndx-nirs~=0.1.0
-
+ndx-nirs~=0.2
```

Comment:
There may be an incompatibility between `pynwb 1.4` (if previously installed, I.e. `Requirement already satisfied: pynwb~=1.4`) and `ndx-nirs 0.2`.

I got the error `ndx-nirs 0.2.0 requires pynwb<2,>=1.5.1, but you'll have pynwb 1.5.0 which is incompatible.` when I ran `pip install -r requirements.txt`. This is easily solved by `pip install pynwb==1.5.1` and `pip install ndx-nirs~=0.2`

Reply:
Hmm, I'm not sure I understand why you got an error. Since ndx-nirs v0.2.0 requires pynwb>=1.5.1, I would have expected it to automatically update pynwb even if you already had a version installed. I tried to reproduce the situation in a local environment, but it always updated pynwb when installing via requirements or by pip install ndx-nirs==0.2.0. 

However, it does seem a bit inconsistent to require pywnb~=1.4 if ndx-nirs v0.2.0 requires pynwb>=1.5.1, so it would be better to update the requirements for pynwb (and probably hdmf) here. I added it to this issue: https://www.pivotaltracker.com/story/show/178567678